griptape/drivers/vector/pgvector_vector_store_driver.py

```diff
@@ -86,15 +87,18 @@ def upsert_vector(
             obj = session.merge(obj)
             session.commit()
 
-            return str(obj.id)
+            return str(getattr(obj, "id"))
 
     def load_entry(self, vector_id: str, namespace: Optional[str] = None) -> BaseVectorStoreDriver.Entry:
         """Retrieves a specific vector entry from the collection based on its identifier and optional namespace."""
         with Session(self.engine) as session:
             result = session.get(self._model, vector_id)
 
             return BaseVectorStoreDriver.Entry(
-                id=result.id, vector=result.vector, namespace=result.namespace, meta=result.meta
+                id=getattr(result, "id"),
+                vector=getattr(result, "vector"),
+                namespace=getattr(result, "namespace"),
+                meta=getattr(result, "meta"),
```

Comment:
What was the error that prompted these changes?

Reply:
Same as above (`Cannot access member "____" for type "_symbol"`) on each of the attributes